src/adaptors/compound-v3/index.js

```diff
@@ -68,6 +92,30 @@ const markets = [
     rewardToken: '0x9e1028F5F1D5eDE59748FFceE5532509976840E0',
     chain: 'base',
   },
+  {
+    address: '0xB2f97c1Bd3bf02f5e74d13f02E3e26F93D77CE44',
+    symbol: 'cUSDCv3',
+    underlying: '0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4',
+    underlyingSymbol: 'USDC',
+    rewardToken: '0x643e160a3C3E2B7eae198f0beB1BfD2441450e86',
+    chain: 'scroll',
+  },
+  {
+    address: '0x2e44e174f7D53F0212823acC11C01A11d58c5bCB',
+    symbol: 'cUSDCv3',
+    underlying: '0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85',
+    underlyingSymbol: 'USDC',
+    rewardToken: '0x7e7d4467112689329f7E06571eD0E8CbAd4910eE',
+    chain: 'optimism',
+  },
+  {
+    address: '0x995E394b8B2437aC8Ce61Ee0bC610D617962B214',
+    symbol: 'cUSDTv3',
```

Comment:
pls add a suffix here, `-2`, tests are currently failing cause of a duplicated id

Reply:
@slasher125 how to update this API? We cannot receive data for Optimism and Scroll COMP data
https://github.com/DefiLlama/yield-server/blob/920f86f0a9c1b16d17e6e98ee0ced6ea6481385c/src/adaptors/compound-v3/index.js#L170

Reply:
And for new markets. Can you help us where should we add it

Reply:
Is this one? https://github.com/DefiLlama/defillama-server/tree/master/coins

Reply:
we have just updated the price data for COMP on both OP and Scroll (https://github.com/DefiLlama/defillama-server/pull/7440). prices will be available ~1h